factory/cluster.go

```diff
@@ -88,29 +96,31 @@ func newCluster(g GinkgoTInterface) (*Cluster, error) {
 }
 
 // DestroyCluster destroys the cluster and returns it
-func DestroyCluster(g GinkgoTInterface) (string, error) {
-	var varDir string
-	cfg, err := shared.EnvConfig()
+func DestroyCluster() (string, error) {
+	cfg, err := config.AddEnv()
 	if err != nil {
 		return "", err
 	}
-	varDir, err = filepath.Abs(shared.BasePath() +
+
+	_, callerFilePath, _, _ := runtime.Caller(0)
+	dir := filepath.Join(filepath.Dir(callerFilePath), "..")
+	varDir, err := filepath.Abs(dir +
```

Comment:
possible to use BasePath function here at all?

Reply:
https://github.com/rancher/distros-test-framework/pull/79#discussion_r1638123337